src/20240602_AaveV3Ethereum_GhoStewardUpdate/GhoStewardUpdate.md

```diff
@@ -0,0 +1,33 @@
+---
+title: "Gho Steward Update"
+author: "ACI"
+discussions: "https://governance.aave.com/t/arfc-gho-stewards/16466/11"
+---
+
+## Simple Summary
+
+This proposal seeks to update the GHO steward to an upgraded version, allowing more granular control over GHO to increase it’s efficiency
+
+## Motivation
+
+The GHO peg is strong, and supply is growing. As GHO matures, new strategies can now be implemented to make it more liquid, resilient and help it grow larger. For example, a strong GSM is expected to allow larger cap increases, allowing faster growth of the supply. Also, in dynamic markets, tighter control on borrow rates is expected to improve asset efficiency.
+
+## Specification
+
+The upgraded Gho Steward can be found on the [gho-core github](https://github.com/aave/gho-core/blob/main/src/contracts/misc/GhoStewardV2.sol)
+
+Changes:
+
+- GSM fee strategy can now be updated by 50 bps both ways
+- GHO Borrow rate can now be updated by 50 bps both ways
```

Comment:
```suggestion
- GHO Borrow cap can now be updated by 50 bps both ways
```

Reply:
also permitted borrow cap change is not 50 bps but the change allowed the current size only.